lib/cloudevents/kafka_binding/v_1_0/decoder.ex

```diff
@@ -15,16 +15,38 @@ defmodule Cloudevents.KafkaBinding.V_1_0.Decoder do
         ) ::
           {:ok, Cloudevents.t()} | {:error, any}
   def from_kafka_message(kafka_body, kafka_headers) do
+    IO.inspect(kafka_headers, label: "from_kafka_message")
+    IO.inspect(content_type(kafka_headers), label: "content_type")
+
     case content_type(kafka_headers) do
-      "application/cloudevents" -> parse_structured(kafka_body, "json")
-      "application/cloudevents+" <> event_format -> parse_structured(kafka_body, event_format)
-      "application/cloudevents-batch" <> _ -> {:error, :batch_mode_not_available_as_per_spec}
-      event_format -> parse_binary(kafka_headers, kafka_body, event_format)
+      "application/cloudevents" ->
+        parse_structured(kafka_body, "json")
+
+      "application/cloudevents+avro" ->
+        parse_binary(kafka_headers, kafka_body, :avro)
```

Comment:
Not happy with this. `parse_binary` is for binary _mode_, just like `parse_structured` is for structured mode. The event format here might be Avro, which is a binary encoding; but the message itself is still _structured mode_.